src/main/java/teammates/common/datatransfer/FeedbackContributionQuestionDetails.java

```diff
@@ -271,8 +271,12 @@ private String getQuestionResultsStatisticsHtmlQuestionView(List<FeedbackRespons
         for(Map.Entry<String, StudentResultSummary> entry : studentResults.entrySet()){
             StudentResultSummary summary = entry.getValue();
             String email = entry.getKey();
-            String name = bundle.roster.getStudentForEmail(email).name;
-            String team = bundle.roster.getStudentForEmail(email).team;
+            StudentAttributes student = bundle.roster.getStudentForEmail(email);
+            if (!bundle.sectionTeamNameTable.containsKey(student.section)) {
```

Comment:
It might be good to change the comment on lines 232 and 332 as well as the method name `getTeamNames` reflect that it only gets team names for teams in sections that are visible to the instructor or in the selected section